book/src/appendices/c_feature_gates.md

```diff
@@ -15,15 +15,50 @@ cargo (build/test/run) --no-default-features --features feature1,feature2,featur
 
 At the time of writing, the list of features of this type is the following:
 
-* animation
-* audio
-* gltf
-* locale
-* network
+* `animation`
+* `audio`
+* `gltf`
+* `locale`
+* `network`
+* `renderer`
+* `saveload`
+* `sdl_controller`
 
 The full list of available features is available in the [Cargo.toml](https://github.com/amethyst/amethyst/blob/master/Cargo.toml) file.
 The available features might change from time to time.
 
+## Graphics features
+
+Whenever you run your game, you'll need enable one graphics backend. The following features are
+available for the backend:
+
+* `empty`
+* `metal`
+* `vulkan`
+
+Rendy has multiple safety checks built-in to detect bugs in the data it gets submitted. However,
+those checks can become too costly for a smooth experience with larger games; you can disable
+them using the `no-slow-safety-checks` feature.
+
+Additionally, there's a `shader-compiler` feature which allows compiling GLSL / HLSL to SPIR-V
+shaders. This is only needed if you're planning to write your own shaders, as Amethyst's built-in 
```

Comment:
Not really just "write your own shaders", but "compile them at runtime". You can always compile the shaders offline and load spirv to the engine, no need for the feature then.